routes/users.js

```diff
@@ -9,23 +9,22 @@ const { upload } = require("../utils/multer");
 const { getUserBadges } = require("../controllers/badges");
 
 router.post("/verify", authenticate, users.verifyUser);
-router.get("/userId/:userId", users.getUserById);
+router.get("/users/:userId", users.getUserById);
```

Comment:
why did we add `/users` this is already present in the `index.js ` file as /users is the base url. @endeavourmonk